Sources/StreamChatUI/VoiceRecording/Components/Dictionary+RawJSONVoiceRecording.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+/// Provides easy setters/getters for the `VoiceRecording` feature required `extraData` fields.
+extension Dictionary where Key == String, Value == RawJSON {
```

Comment:
Shouldn't this be a private extension where it is used? IMO this should not be internally exposed to the SDK

Reply:
If this is being made `private` we can't access and we are using it from a few places (so we can't really move it in the file that is being used from). Are you too concerned about it? It's not really affecting the behaviour of the dictionary just allowing for easier access.

Reply:
Because this is our "data" this shouldn't actually be parsed as extra data 🤔 Since we have a `VoiceRecordingAttachmentPayload`, the `duration` and `waveform` properties should be part of the payload itself. Any reason it isn't this way?

Reply:
That is a very good suggestion. I (mistakenly obviously) thought that in order to change the Payload we require BE work. I have now updated this to used the payload data and not extraData. Thanks :)